app/utils/transaction-view.js

```diff
@@ -193,3 +192,13 @@ function hasPageSizeLinks (connectorData) {
   const paginator = new Paginator(connectorData.total, getCurrentPageSize(connectorData), getCurrentPageNumber(connectorData))
   return paginator.showDisplaySizeLinks()
 }
+
+function showCsvDownload (connectorData, filters) {
+  if (connectorData.total <= LEDGER_TRANSACTION_COUNT_LIMIT) {
+    return true
+  }
+
+  return Object.keys(filters).filter(function (key) {
+    return key !== 'page' && key !== 'pageSize'
+  }).length !== 0
```

Comment:
Very minor but this is probably sufficiently complex that naming it would help readability

```javascript
const hasUserFilteredResults = Object.keys(filters)
  .filter((key) => !['page', 'pageSize'].includes(key))
  .length

return hasUserFilteredResults
```

Reply:
Done this and IDE complained to make it inline 😄

Reply:
Updated in https://github.com/alphagov/pay-selfservice/pull/2259/files#diff-bc5732d3a5d7c7ae7ed3dfa0a9893fb2R201